pkg/brokerapi/openservicebroker/open_service_broker_client.go

```diff
@@ -40,9 +40,10 @@ const (
 	serviceInstanceAsyncFormatString       = "%s/v2/service_instances/%s?accepts_incomplete=true"
 	serviceInstanceDeleteFormatString      = "%s/v2/service_instances/%s?service_id=%s&plan_id=%s"
 	serviceInstanceDeleteAsyncFormatString = "%s/v2/service_instances/%s?service_id=%s&plan_id=%s&accepts_incomplete=true"
-	pollingFormatString                    = "%s/v2/service_instances/%s/last_operation"
+	pollingFormatString                    = "%s/v2/service_instances/%s/last_operation?%s"
```

Comment:
:+1: including the operation as part of the string. What happens if the provision response does NOT return an operation?  Will the dangling '?' be a problem?